### DISCUSSION TITLE
integrazione centraline aqicn

### DISCUSSION THREAD

**Discussion by Michelone75** [indomus/forum#1716] [GENERALE - Discussioni generiche]
Ho seguito il vostro articolo inerente l'integrazione delle centraline aqicn, ma non riesco ad inserire la centralina a me vicina. il codice che inserisco è il seguente

  - platform: waqi
    token: 03dc76-----------------------------------e9460
    locations:
      - Via Vittorio Veneto, Cassina de' Pecchi, Italy
 
ma una volta riavviato HA l'entità waqi ha dento una serie di opzioni tranne quella che serve a me, cosa sbaglio?

la centralina che ho provato ad aggiungere è questa

https://aqicn.org/station/@96481/

Grazie per l'aiuto
MICHELE

**Comment by Michelone75** [comment]
ah dimenticavo,,,inserendo poi sotto il codice 

  - platform: template   
    sensors:
      aqicn:
        friendly_name: "Qualità dell'aria di Roma" #campo customizzabile
        value_template: >-
          {% if states('sensor.waqi_lecco_via_sora_lombardia_italy') | int(default=0) <= 50 %}
          Buona
          {% elif states('sensor.waqi_lecco_via_sora_lombardia_italy') | int(default=0) <= 100 %}
          Moderata
          {% elif states('sensor.waqi_lecco_via_sora_lombardia_italy') | int(default=0) <= 150 %}
          Insalubre per alcuni
          {% elif states('sensor.waqi_lecco_via_sora_lombardia_italy') | int(default=0) <= 200 %}
          Insalubre
          {% elif states('sensor.waqi_lecco_via_sora_lombardia_italy') | int(default=0) <= 300 %} 
          Molto insalubre 
          {% elif states('sensor.waqi_lecco_via_sora_lombardia_italy') | int(default=0) >= 300 %}
          PERICOLOSA
          {% endif %}%}

non mi crea il sensor.aqicn e non capisco perchè :(

**Comment by KC-inDomus** [comment]
prova a mettere "Via Vittorio Veneto, Cassina de' Pecchi, Milano, Italy"
